packages/tools/examples/PolySegWasmContourToSurface/index.ts

```diff
@@ -150,6 +152,21 @@ addButtonToToolbar({
   },
 });
 
+addLabelToToolbar({
```

Comment:
Can you add a helper method for this bit of code?  It feels like it will be jsut common enough to be useful having.